solutions/bronze/usaco-916.mdx

```diff
@@ -4,11 +4,22 @@ source: USACO Bronze 2019 February
 title: The Great Revegetation
 author: Ryan Chou, Brad Ma
 ---
-## Solution 1
+
+<Spoiler title="Hint">
+
+What's the best case scenario, and what forces us to change to a different grass type?
```

Comment:
wdym "best case scenario"

Reply:
a case where everyone can share the same grass, so you only have 1111...

Reply:
yeah, imo the first part (before the comma) isn't rlly necessary?

Reply:
ok bet